build.gradle.kts

```
@@ -69,6 +69,50 @@ tasks.register("publishMiraiCoreArtifactsToMavenLocal") {
     )
 }
 
+tasks.register("verifyCompiledClasses") {
+    val projects = listOf("mirai-core-api", "mirai-core-utils", "mirai-console", "mirai-console-terminal")
+    group = "verification"
+    doFirst {
+        projects.asSequence().mapNotNull {
+            runCatching { project(it) }.getOrNull()
+        }.map { project ->
+            val classesDir = project.buildDir.resolve("classes")
+            listOf(
+                listOf("kotlin/main", "kotlin/jvm/main") to listOf("runtimeClasspath", "jvmRuntimeClasspath"),
+                listOf("kotlin/main", "kotlin/android/main") to listOf(
+                    "runtimeClasspath",
+                    "androidRuntimeClasspath"
+                )
+            ).map { (targetClasses, classpath) ->
+                val libs = classpath.asSequence().mapNotNull {
+                    kotlin.runCatching {
+                        project.configurations.getByName(it)
+                    }.getOrNull()
+                }.map { it.files.asSequence() }.flatten()
```

```suggestion
                    val libs = classpath.flatMap {
                        project.configurations.findByName(it)?.files ?: emptyList()
                    }
```

- `asSequence`很多地方没有必要，不明白为啥要加
- `flatMap` API请了解一下
- `NamedObjectCollection.findByName` API请了解一下